shared/data-structures/linked-list.js

```diff
@@ -123,9 +123,9 @@ var List = Interface.define("List", {
       var node = this.head;
       var success = false;
       return (function() {
-        var while$5 = undefined;
+        var while$9 = undefined;
```

Comment:
This kinda stuff is just noise from the sibilant transpiler. It probably happens because the files are getting compiled in different orders every time.

Reply:
Added a task for this.